packages/dts-gen/src/index.ts

```diff
@@ -34,4 +34,8 @@ const handler = async () => {
   await TypeDefinitionTemplate.renderAsFile(args.output, input);
 };
 
-handler().catch((err) => console.error(err));
+handler().catch((err) => {
+  console.error(err);
+  // eslint-disable-next-line no-process-exit
+  process.exit(2);
```

Comment:
```suggestion
  process.exit(1);
```

We need more time to consider the return values for each error.
Please keep using `1` as the exit status for all failures at this time.

Reply:
@mshrtsr 
Thanks for the review.
I comitted your suggestion!